### DISCUSSION TITLE
Using Tailwind Classes in PT mode

### DISCUSSION THREAD

**Discussion by rogrile** [primefaces/community#497] [PrimeVue]
Hey, I am currently using PT with Nuxt3.
I have created a folder inside assets/presets/ where I've added my preset. Here I just expand on the tailwind passthrough preset that PrimeVue comes with.

It works as in I have my styles working in most cases. Buut, I want to add for example a NuxtLink with the class 'p-button' which does not work.

You can check it out here (this is the config that shows how 'hover:' isn't working properly with PT as well):
https://stackblitz.com/edit/nuxt-starter-tfyadx?file=assets%2Fpresets%2Fgrilero%2Findex.js

**Comment by tugcekucukoglu** [comment]
Hi,

Unstyled mode removes all styles that come from PrimeVue theme. So, you can't find 'p-button' style class. Also, Tailwind scale classes use transform, and when these classes are given directly to the root, the Button will not transform itself and its children.

**Reply by rogrile** [reply]
Thanks for the answer. So if I go with styled and tailwindcss, but using Tailwind PT, this will not work as these classes are not generate through the PT Tailwind config?

**Reply by rogrile** [reply]
Also, if this is the case with scaling and transform, how can one achieve this though? For example, I tried modifying to get a sticky menubar, but it is not working although I just added the fixed top-0 left-0 absolute (and relative to the parent).
